openc3/Dockerfile

```diff
@@ -24,9 +24,11 @@ RUN mkdir -p lib/openc3/ext \
   && mkdir -p gems \
   && mv *.gem gems/. \
   # TODO: Remove google-protobuf / grpc manually install once grpc fixed
+  # Uninstall twice is intentional because --all doesn't work
```

Comment:
I removed the previous double install because I thought it was a typo. Thanks for the comment! Hopefully this all goes away soon.